src/Http/Authentication.Abstractions/src/AuthenticationTicket.cs

```diff
@@ -55,8 +55,10 @@ public AuthenticationTicket(ClaimsPrincipal principal, string authenticationSche
 
         /// <summary>
         /// Returns a copy of the ticket.
-        /// Note: the claims principal will be cloned by calling Clone() on each of the Identities.
         /// </summary>
+        /// <remarks>
+        /// The method clones the <see cref="Principal"/> by calling <see cref="ClaimsIdentity.Clone"/> on each of the <see cref="ClaimsPrincipal.Identities"/>.
```

Comment:
`The <see cref="Principal"/> is cloned by calling <see cref="ClaimsIdentity.Clone"/> on each of the <see cref="ClaimsPrincipal.Identities"/>.`